include/admin.jsonnet

```diff
@@ -85,7 +85,16 @@ local common = import "common.jsonnet";
                       {
                         "name": "CONFIGURATION_SERVICE_PORT",
                         "value": "5672"
-                      }]),
+                      },
+                      {
+                        "name": "API_SERVER_HOSTNAME",
```

Comment:
Minor suggestion on naming... perhaps ADDRESS_CONTROLLER_HOST would be more obvious? Or else ADDRESS_SERVICE_HOST (to emulate loosely the kubernetes naming convention)?